README.AnyChart.md

```diff
@@ -32,6 +32,8 @@ Static : `[ { "type": "scatter" } ]` and Sample data `[ { "x": [ 1, 2 ], "y": [
 
 ![Any Chart Configuration](/assets/cheatsheet/AnyChartConfiguration.png)
 
+Making chart from cratch is not an easy task. This guide helps you get started. Also have a look at https://chart-studio.plot.ly/create this allows to create a chart visually and peek the JSON output from the left menu bottom section.
```

Comment:
"Making charts from scratch..."

Reply:
Also, don't forget to mention that the widget supports charts supported by plotly v1.51